scripts/measures/lib/merge-ecqm-data.ts

```diff
@@ -0,0 +1,22 @@
+export function mergeEcqmData (measures, generatedEcqms) {
+  generatedEcqms.forEach((generatedEcqm) => {
+    const measure = measures.find(measure => measure.eMeasureId === generatedEcqm.eMeasureId);
+    if (measure) {
+      if (measure.strata) {
```

Comment:
Could we combine these lines, `if (measure?.strata)`?

Reply:
I don't think we can, since lines 17-19 need to run if measure exists, even if measure.strata does not.